tests/adapter/dbt/tests/adapter/incremental/fixtures.py

```diff
@@ -361,35 +303,3 @@
 
 from source_data
 """
-
-#
-# Tests
-#
-
-_TESTS__SELECT_FROM_INCREMENTAL_IGNORE = """
```

Comment:
Clearly from my more naive days of test conversions :p 

That said, I do recall thinking "should we keep these?" I know they live in the code history and don't necessary need to exist in current code. More a "principles" question: if there are seemingly extraneously models in tests, do we nix them with cautious abandon or seek to add tests to include them?

Reply:
As a general rule: If it's not being actively used in one of our existing tests, we probably don't need it. (There are rare exceptions, where we've added a new fixture/combination with the _intent_ of testing, but completely forgot! I just found an instance where I'd made this mistake a ~while ago: https://github.com/dbt-labs/dbt-bigquery/pull/41)

Longer answer: Intent is a useful input, to the extent we can infer it. In this case, I remember the history as being: our colleague Matt W added this feature (`on_schema_change`) back in 2021, and he added integration tests for it by copying + adapting some other recent integration tests (for `--store-failures`). The "tests" component was important for the original tests, but not necessary for these adapted ones.

Reply:
Appreciate the history lesson! That makes much more sense and I'll run with your heuristic in the future.